src/multicellular_stage/editor/CellBodyPlanEditorComponent.GUI.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using Godot;
+using Systems;
+
+/// <summary>
+///   The partial class containing GUI updating actions
+/// </summary>
+public partial class CellBodyPlanEditorComponent
+{
+    protected override void OnTranslationsChanged()
+    {
+        organismStatisticsPanel.OnTranslationsChanged();
+    }
+
+    private void ConfirmFinishEditingWithNegativeATPPressed()
+    {
+        if (OnFinish == null)
+        {
+            GD.PrintErr("Confirmed editing for cell editor when finish callback is not set");
+            return;
+        }
+
+        GUICommon.Instance.PlayButtonPressSound();
+
+        ignoredEditorWarnings.Add(EditorUserOverride.NotProducingEnoughATP);
+        OnFinish.Invoke(ignoredEditorWarnings);
+    }
+
+    private void HandleProcessList(IReadOnlyList<HexWithData<CellTemplate>> cells, EnergyBalanceInfo energyBalance,
+        IBiomeConditions biome)
+    {
+        // Empty list to later fill
+        var processStatistics = new List<ProcessSpeedInformation>();
+
+        var processes = new List<TweakedProcess>();
+
+        foreach (var cellType in GetCellTypes())
+        {
+            var newProcesses = new List<TweakedProcess>();
```

Comment:
This list should be moved outside the loop so that it can be reused in each loop iteration by just clearing it. This will ever so slightly save on memory allocations.